osidb/urls.py

```diff
@@ -21,12 +23,16 @@
 router.register(r"affects", AffectView)
 router.register(r"trackers", TrackerView)
 
+nested_router = NestedDefaultRouter(router, r"flaws", lookup="flaw")
```

Comment:
This is interesting, but given that it's already possible to create the same behavior with vanilla DRF [1] I'd prefer not to add this dependency

[1] https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing

Reply:
I have not been able to get all `@action` working yet, so at least I removed the external library and updated the router directly.

When manually running the command to generate openapi schema, l get only this warning: `Warning #8: FlawReferenceView: could not derive type of path parameter "id" because model "osidb.models.FlawReference" contained no such field. Consider annotating parameter with @extend_schema. Defaulting to "string".`